src/platform_impl/macos/app_state.rs

```diff
@@ -282,33 +325,20 @@ impl ApplicationDelegate {
     }
 
     #[track_caller]
-    fn maybe_queue_event(&self, event: Event<HandlePendingUserEvents>) {
```

Comment:
Nit: I'd like to keep this as something like:
```rust
fn maybe_queue_with_user_app(&self, closure: impl FnOnce(&mut dyn ApplicationHandler, &RootActiveEventLoop) + 'static) { ... }
```

Instead of duplicating it across `maybe_queue_window_event` and `maybe_queue_device_event`.